packages/desktop-client/src/components/modals/EditRule.js

```diff
@@ -91,17 +91,23 @@ export function OpSelect({
   formatOp = friendlyOp,
   onChange,
 }) {
+  let line;
   // We don't support the `contains` operator for the id type for
   // rules yet
   if (type === 'id') {
-    ops = ops.filter(op => op !== 'contains');
+    ops = ops.filter(op => op !== 'contains' && op !== 'doesNotContain');
+    line = ops.length / 2;
+  }
+  if (type === 'string') {
+    line = ops.length / 2;
   }
 
   return (
     <Select
       options={ops.map(op => [op, formatOp(op, type)])}
       value={value}
       onChange={value => onChange('op', value)}
+      line={line}
```

Comment:
Does this do anything? I don’t see any effect from this.

Reply:
Ahh, yes. I had changes made to CustomSelect but that's been removed in a previous merge so it removed my changes. I've re-added the changes. It just allows for a number to be passed to the element and have the element insert a line after the position of that array. 

This is another look/feel change that can be reverted. I thought it looked better with the options being divided in the menu.

Reply:
That makes a ton of sense! I remember liking that from your earlier screenshots. Thanks for restoring the functionality!